pkgs/top-level/all-packages.nix

```diff
@@ -6787,6 +6787,8 @@ with pkgs;
   oraclejdk10distro = packageType: _:
       (callPackage ../development/compilers/oraclejdk/jdk10-linux.nix { inherit packageType; });
 
+  javacard-devkit = pkgsi686Linux.callPackage ../development/compilers/javacard-devkit { };
```

Comment:
Why is it `pkgsi686Linux`?

Reply:
The devkit is available only for 32-bits linux, and includes a pre-compiled executable (`cref`)

Reply:
Why is `pkgsi686Linux` used in stead of `meta.platforms`? Does it have something to do with cross compiling?

Reply:
The devkit does work on 64-bit linux, but requires 32-bit libraries, that are defined by `pkgsi686Linux` :) That said, you're right, I had mis-set `meta.platforms`! This should be fixed with latest push :)